skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +325,67 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom(self):
```

Comment:
Can we split this into a couple test cases, each one tackling a specific success or failure condition?